scipy/signal/filter_design.py

```diff
@@ -299,12 +304,15 @@ def freqz(b, a=1, worN=None, whole=False, plot=None):
         A callable that takes two arguments. If given, the return parameters
         `w` and `h` are passed to plot. Useful for plotting the frequency
         response inside `freqz`.
+    fs : float
+        The sampling frequency of the digital system.
+        .. versionadded:: 1.1.0
 
     Returns
     -------
     w : ndarray
-        The normalized frequencies at which `h` was computed, in
-        radians/sample.
+        The frequencies at which `h` was computed.  If `fs` is specified,
+        these are in the same units.  Otherwise they are in radians/sample.
```

Comment:
This is a case where setting `fs=2*pi` makes things simpler. Here you can just say they are in units of the sample rate, and you can get rid of the conditionals below. We can make `fs=None` behave like `2 * pi` if this would make achieving "use-the-default" behavior easier for people. @WarrenWeckesser WDYT?

Reply:
The `worN=None` arguments follow the same pattern, right?  They actually default to numbers, too.

Reply:
Yes, I am arguing against that design choice as it hurts argument introspection. I think we should ideally change this everywhere, so at least avoid propagating the problem here.

Reply:
I just remembered this was discussed in #7788. I'll try to move that forward so we can proceed here.

Reply:
Maybe that should be done in a separate PR then?  (And changing `worN=512` as well?)

Reply:
Sorry for disappearing from this conversation.  If no one else objects to `fs=2*np.pi`, then go ahead.  (Of course, introspection of the signature will then show `fs=6.283185307179586`.)

Reply:
That's a neat idea, but I don't think the extra code is worth it just to fix a cosmetic issue.

Reply:
Back to this line of code -- we should just say that the frequencies are in units of `fs` (and if you want, add that the default `fs=2*pi` implies the units are radians/sample)